style/js/shield_defs.js

```diff
@@ -2279,26 +2280,41 @@ export function loadShields(shieldImages) {
     2,
     1
   );
-
+  let nlCityRoute = {
+    backgroundImage: shieldImages.shield40_nl_city,
+    textColor: Color.shields.black,
+    padding: {
+      left: 3,
+      right: 3,
+      top: 3,
+      bottom: 3,
+    },
+  };
   [
     "Amsterdam",
     "Den Haag",
-    "Rotterdam",
     "Nijmegen",
     "Parkstad",
+    "Rotterdam",
     "Zaanstad",
+  ].forEach((city) => (shields[`NL:S:${city}`] = nlCityRoute));
+  shields["NL:binnenstedelijke_ring"] = nlCityRoute; // for both Netherlands and Curacao
+  [
+    "Ommen",
+    "Schouwen",
+    "Sluis",
+    "Spaarnwoude",
+    "Voorthuizen",
+    "IJmuiden",
   ].forEach(
-    (city) =>
-      (shields["NL:S:" + city] = {
-        backgroundImage: shieldImages.shield40_nl_city,
-        textColor: Color.shields.black,
-        padding: {
-          left: 3,
-          right: 3,
-          top: 4,
-          bottom: 5,
-        },
-      })
+    (place) =>
+      (shields[`NL:R:${place}`] = roundedRectShield(
+        Color.shields.brown,
+        Color.shields.white,
+        Color.shields.white,
+        2,
+        1
```

Comment:
With #355 now merged, only the first two arguments of `roundedRectShield()` are necessary, and the rest can be omitted.

Reply:
Great! Fixed in merge commit.

Reply:
Great! Fixed in merge commit.